### PR TITLE
Checking 681

### LINKED ISSUE #681: Niceify check-function
The check-function has some aesthetical shortcomings:
- result is printed on next line
- it stops on first error

### PR DESCRIPTION
**Pull request by ineiti**
Closes #681

### CODE REVIEW

File: app/lib/server/server.go [hunk 1]
```diff
@@ -255,7 +256,7 @@ func checkList(list *sda.Roster, descs []string) error {
 	}
 	log.Lvl3("Sending message to: " + serverStr)
 	msg := "verification"
-	log.Info("Checking server(s) ", serverStr, ": ")
+	fmt.Printf("Checking server(s) %s: ", serverStr)
```
Comment (nikkolasg) [inline app/lib/server/server.go]: why going back  to fmt ?

Comment (nikkolasg) [inline app/lib/server/server.go]: What about having a `log.Infof` ? (that does not add a "\n" unless you explicitely put)

Comment (ineiti) [inline app/lib/server/server.go]: All `log.*f`-methods add a "\n", so this would be the exception.

**Comment by nikkolasg** [comment]
Looks good to me. Merging.

**Pull request merged**